pkg/commands/patch/patch_modifier.go

```diff
@@ -44,6 +46,10 @@ func GetHunksFromDiff(diff string) []*PatchHunk {
 			continue
 		}
 
+		if lineIdx == len(lines)-1 && line == "" { // skip the trailing newline
```

Comment:
I noticed that you can actually select past the bottom of the patch thank to a trailing newline that we treat as a separate line in its own right. I've got a couple of changes in this PR that trim that trailing newline, and from all my testing it seems like a safe change. We still ensure that upon copying the patch to the clipboard it's got a trailing newline in it.